solutions/4.cpp

```diff
@@ -1,3 +1,20 @@
+<<<<<<< HEAD
```

Comment:
looks like merge conflict not handled properly

Reply:
yeah. ill delete those lines really quickly